cmd/really/really.cpp

```diff
@@ -164,23 +152,24 @@ class ReallyServerDelegate : public quicr::ServerDelegate
   {
   }
 
-  void onPublisherObject(const qtransport::TransportContextId& context_id,
-                         [[maybe_unused]] const qtransport::StreamId& stream_id,
-                         [[maybe_unused]] bool use_reliable_transport,
-                         quicr::messages::PublishDatagram&& datagram) override
+  std::vector<quicr::PublishResult> onPublisherObject(
+    const qtransport::TransportContextId& context_id,
+    [[maybe_unused]] const qtransport::StreamId& stream_id,
+    [[maybe_unused]] bool use_reliable_transport,
+    quicr::messages::PublishDatagram&& datagram) override
   {
-    const auto list = subscribeList.find(datagram.header.name);
-
-    for (auto dest : list) {
-      if (dest.context_id == context_id) {
-        // split horizon - drop packets back to the source that originated the
-        // published object
-        continue;
-      }
-
-      // TODO(trigaux): Move logic into quicr::Server
-      server->sendNamedObject(dest.subscribe_id, false, 1, 200, datagram);
+    auto list = subscribeList.find(datagram.header.name);
+
```

Comment:
I had added dedup here, but after testing with Decimus I removed it as it only really hindered testing. Joining a Team meeting is fine, you won't receive yourself because you didn't subscribe to it. Mirror calls work very nicely.